cinema/serializers.py

```diff
@@ -44,14 +57,16 @@ class MovieListSerializer(MovieSerializer):
         many=True, read_only=True, slug_field="full_name"
     )
 
+    class Meta(MovieSerializer.Meta):
+        fields = MovieSerializer.Meta.fields + ("image",)
+
 
 class MovieDetailSerializer(MovieSerializer):
     genres = GenreSerializer(many=True, read_only=True)
     actors = ActorSerializer(many=True, read_only=True)
 
-    class Meta:
-        model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+    class Meta(MovieSerializer.Meta):
+        fields = MovieSerializer.Meta.fields + ("image",)
```

Comment:
I'm not sure, but if we use this in both: MovieDetail and MovieList serializer classes, maybe it is worth moving the "image" field to MovieSerializer fields? to the row 42
This is an assumption, to change it or not is at your discretion)